index.js

```diff
@@ -167,19 +164,17 @@ module.exports = function (app) {
               else {
                 writeApi.writePoint(influxFormat(seperatePath.path,seperatePath.value,seperatePath.timestamp,options))
               }
-              
             })
           }
           else {
-            if (isNaN(values)) {
+            if (isNaN(values) || !isfloatField(values)) {
```

Comment:
I came across an issue with the derived data plugin (Fuel Economy) Speed/Rate. When fuel rate is 0 we get an infinity value what if we change this line to?
`if (isNaN(values) || !isfloatField(values) || !isFinite(values))`